documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -83,39 +85,19 @@ See xref:con-versions-and-images-str[Kafka version and image mappings]
 
 . Save and exit the editor, then wait for rolling updates to complete.
 +
-NOTE: Additional rolling updates occur if the new version of Kafka has a new ZooKeeper version.
-+
-Check the update in the logs or by watching the pod state transitions:
-+
-[source,shell,subs=+quotes]
-----
-kubectl logs -f _CLUSTER-OPERATOR-POD-NAME_ | grep -E "Kafka version upgrade from KafkaVersion.*[0-9.]+.* to KafkaVersion.*[0-9.]+.*completed"
-----
+Check the progress of the rolling updates by watching the pod state transitions:
 +
 [source,shell,subs=+quotes]
 ----
 kubectl get pods my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
 ----
 +
-If the current and new versions of Kafka have different inter-broker protocol versions, check the Cluster Operator logs for an `INFO` level message:
-+
-[source,shell,subs=+quotes]
-----
-Reconciliation #_NUM_(watch) Kafka(_NAMESPACE_/_NAME_): Kafka version upgrade from _FROM-VERSION_ to _TO-VERSION_, phase 2 of 2 completed
-----
-Alternatively, if the current and new versions of Kafka have the same interbroker protocol version, check for:
-+
-[source,shell,subs=+quotes]
-----
-Reconciliation #_NUM_(watch) Kafka(_NAMESPACE_/_NAME_): Kafka version upgrade from _FROM-VERSION_ to _TO-VERSION_, phase 1 of 1 completed
-----
-+
 The rolling updates:
 +
 * Ensure each pod is using the broker binaries for the new version of Kafka
-* Configure the brokers to send messages using the inter-broker protocol of the new version of Kafka
+* Configure the brokers to send messages using the inter-broker protocol version of the new version of Kafka
 +
-NOTE: Clients are still using the old version, so brokers will convert messages to the old version before sending them to the clients. To minimize this additional load, update the clients as quickly as possible.
+IMPORTANT: Clients are still using the old Kafka version, so brokers will convert messages to the old version before sending them to the clients. To minimize this additional load, update all client applications as soon as possible.
```

Comment:
I think this is misleading. It's not the client version itself that matters. Message conversion can be caused by different message format between producer and topic (which, yes defaults to log.message.format.version, but can be overridden per topic), but also things like mismatched compression. Can we remove this? I think the following step makes it a bit clearer, since the strategies are explained in a bit more detail.

Reply:
Removed the note.